server/config/routes.js

```diff
@@ -25,7 +25,8 @@ module.exports.routes = {
   "get /api/sermons/get-sermon-by-id": "sermons/get-sermon-by-id",
   "get /api/sermons/get-speaker": "sermons/get-speaker",
   "get /api/sermons/get-sermon-series": "sermons/get-sermon-series",
-  "post /api/users/add-user": "users/add-user",
+  
+  "get /api/users/view-user": "users/view-user",
```

Comment:
can we keep the naming convention consistent? since all of our other APIs are called `get*`, maybe we can also have `get-user`? unless there's a specific reason why we are using `view-user` that i'm not understanding

Reply:
not really any specific reason. i can change it to get-user